interactive-calculator/src/App.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import './App.css';
+
+class App extends Component {
+  state = {
+    total: "",
+    service:'',
+    partySize: '',
+    totalTip: '',
+    tipPP: ''
+  }
+
+  handleTotal = e => {
+    this.setState({
+      total: e.target.value
+    })
+  }
+  handleService = e => {
+    this.setState({
+      service: e.target.value
+    })
+  }
+  handleParty = e => {
+    this.setState({
+      partySize: e.target.value
+    })
+  }
+  genTip = e => {
+    e.preventDefault()
+    let tip = parseFloat(this.state.total * this.state.service).toFixed(2)
+    let totalTipPP = parseFloat(tip / this.state.partySize).toFixed(2)
+    this.setState({
+      totalTip: tip,
+      tipPP: totalTipPP
```

Comment:
rename variable totalTipPP to tipPP  and totalTip to tip so you could just do this:

```javascript
   this.setState({
      tip,
      tipPP
  })
```
it is cleaner and less variables to worry about.